app/chitter.rb

```diff
@@ -0,0 +1,86 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require 'sinatra/partial'
+
+require_relative 'app_helpers'
+require_relative 'data_mapper_setup'
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+  set :session_secret, 'super secret'
+  register Sinatra::Flash
+  register Sinatra::Partial
+  set :partial_template_engine, :erb
+  helpers ChitterHelpers
+
+  get '/' do
+    @topbox = current_user ? :peep_form : :welcome
+    @peeps = Peep.all
+    erb :'index'
+  end
+
+  get '/sign-up' do
+    @topbox = :user_sign_up
+    @peeps = Peep.all
+    erb :index
+end
```

Comment:
Lint/BlockAlignment: `end` at 27, 0 is not aligned with `get '/sign-up' do` at 23, 2